uppsrc/plugin/zip/zip.h

```diff
@@ -125,8 +127,8 @@ class Zip {
 public:
 	Callback WhenError;
 
-	void BeginFile(const char *path, Time tm = GetSysTime(), bool deflate = true);
-	void BeginFile(OutFilterStream& oz, const char *path, Time tm = GetSysTime(), bool deflate = true);
+	void BeginFile(const char *path, Time tm = GetSysTime(), bool deflate = true, bool zip64 = false);
+	void BeginFile(OutFilterStream& oz, const char *path, Time tm = GetSysTime(), bool deflate = true, bool zip64 = false);
 	void Put(const void *data, int size);

```

Comment:
Good work overall, I am sort of missing Put(... int64 size).

Reply:
OK, I will see to it... probably add Put64(... int64 size).
// Tom

Reply:
* Do not merge! * I have broken it on some zip64 header offset. I will investigate and fix it and let you know when it's OK again.

Reply:
OK, now I have fixed the offset issue when creating large archives with many files and file sizes. Also, I have added Put64(... int64 size). Is this OK, now?
// Tom